src/ChannelInterface.php

```diff
@@ -7,4 +7,6 @@
 interface ChannelInterface extends ConfigEntityInterface {
   public function getName();
   public function getUrl();
+  public function getContent();
```

Comment:
Could these be named Description, Instructions etc. instead of content to clarify the purpose?